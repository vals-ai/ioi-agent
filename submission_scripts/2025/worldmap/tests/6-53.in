version https://git-lfs.github.com/spec/v1
oid sha256:8792511558ac26acc5e3f478166fc31fb71b482394a8d29f8223b7614e14edc5
size 96636
