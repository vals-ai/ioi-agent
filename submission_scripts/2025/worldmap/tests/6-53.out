version https://git-lfs.github.com/spec/v1
oid sha256:0a729c0641dd424799f54e01286d175f715d8a1ec5a76c41e4c1d1b53553f06c
size 720761
